lib/graphql/schema.flow.js

```diff
@@ -14,7 +14,7 @@ export type Scalars = {|
 
 /** The bank account of the current user */
 export type Account = {|
-   __typename?: 'Account',
```

Comment:
We still have a number of apparently unrelated changes here 🤔 But if you had them when running the `schema:generate` command, all good

Reply:
yes, that's what it generated 🤷‍♂️